### PR TITLE
Update support response time content

### PR DESCRIPTION
**Pull request by karlchillmaid**
This PR updates all mentions of our support response times to make the information consistent.

### CODE REVIEW

File: app/templates/views/support/index.html [hunk 1]
```diff
@@ -10,9 +10,10 @@
 {% block maincolumn_content %}
 
    {{ govuk_page_header("Support") }}
-  <p class="govuk-body">GOV.UK Notify provides 24-hour online support.</p>
-  <p class="govuk-body">If it’s an emergency, we’ll reply within 30 minutes and update you every hour until we fix the problem.</p>
-<p class="govuk-body">If it’s not an emergency, we’ll reply within 1 working day.</p>
+  <p class="govuk-body">GOV.UK Notify provides 24/7 support for emergencies.</p>
+  <p class="govuk-body">If you have an emergency, we’ll reply within 30 minutes.</p>
+  <p class="govuk-body">If it’s not an emergency, we’ll reply by the end of the next working day.</p>
+  <p class="govuk-body">Our working days are Monday to Friday.</p>
```
Comment (karlchillmaid) [inline app/templates/views/support/index.html:16]:
```suggestion
  <p class="govuk-body">Our working days are Monday to Friday, 9:30am to 5:30pm, excluding bank holidays.</p>
```

File: app/templates/views/support/thanks.html [hunk 2]
```diff
@@ -15,20 +15,19 @@
...
-            We’ll aim to read your message in the next 30 minutes and we’ll reply within one
-            working day.
+            We’ll reply by the end of the next working day.
           {% endif %}
         {% else %}
           {% if out_of_hours %}
             We’ll read your message when we’re back in the office.
           {% else %}
-            We’ll aim to read your message in the next 30 minutes.
+            We’ll reply by the end of the next working day.
```
Comment (karlchillmaid) [inline app/templates/views/support/thanks.html:30]: I wonder if this logic can be simplified, as a lot of the content is repeated.

Comment (klssmith) [reply app/templates/views/support/thanks.html]: I've made a change to simplify the logic, but I think someone else will need to approve now

File: app/templates/views/support/index.html [hunk 3]
```diff
@@ -10,9 +10,10 @@
 {% block maincolumn_content %}
 
    {{ govuk_page_header("Support") }}
-  <p class="govuk-body">GOV.UK Notify provides 24-hour online support.</p>
-  <p class="govuk-body">If it’s an emergency, we’ll reply within 30 minutes and update you every hour until we fix the problem.</p>
-<p class="govuk-body">If it’s not an emergency, we’ll reply within 1 working day.</p>
+  <p class="govuk-body">GOV.UK Notify provides 24/7 support for emergencies.</p>
+  <p class="govuk-body">If you have an emergency, we’ll reply within 30 minutes.</p>
+  <p class="govuk-body">If it’s not an emergency, we’ll reply by the end of the next working day.</p>
+  <p class="govuk-body">Our working days are Monday to Friday, 9:30am to 5:30pm, excluding bank holidays.</p>
```
Comment (karlchillmaid) [inline app/templates/views/support/index.html:16]:
```suggestion
  <p class="govuk-body">GOV.UK Notify provides 24/7 support for emergencies.</p>
  <p class="govuk-body">If you have an emergency we’ll reply within 30 minutes.</p>
  <p class="govuk-body">If it’s not an emergency we’ll reply by the end of the next working day.</p>
  <p class="govuk-body">Our working days are Monday to Friday, 9:30am to 5:30pm, excluding bank holidays.</p>
```

**Review by CrystalPea — APPROVED** [review]

**Pull request merged**